components/feature/addons/src/main/res/values/strings.xml

```diff
@@ -7,6 +7,18 @@
     <string name="mozac_feature_addons_permissions_privacy_description">Read and modify privacy settings</string>
     <!-- Description for all_urls add-on permission. -->
     <string name="mozac_feature_addons_permissions_all_urls_description">Access your data for all websites</string>
+    <!-- Description for one site access add-on permission. %1$s  will be replaced by the DNS host name for which a web extension is requesting access (e.g., www.mozilla.org). -->
+    <string name="mozac_feature_addons_permissions_one_site_description">Access your data for %1$s</string>
+    <!-- Description for accessing multiple sites in one domain add-on permission. %1$s  will be replaced by the DNS domain for which a web extension is requesting access (e.g., mozilla.org). -->
+    <string name="mozac_feature_addons_permissions_sites_in_domain_description">Access your data for sites in the %1$s domain</string>
+    <!-- Description for accessing multiple sites permission. %1$d will be replaced by an integer indicating the number of additional hosts for which this webextension is requesting permission. -->
+    <string name="mozac_feature_addons_permissions_to_many_sites_description" tools:ignore="PluralsCandidate">Access your data on %1$d other site</string>
+    <!-- Description for accessing multiple sites permission. %1$d will be replaced by an integer indicating the number of additional hosts for which this webextension is requesting permission. -->
+    <string name="mozac_feature_addons_permissions_to_many_sites_description_plural" tools:ignore="PluralsCandidate">Access your data on %1$d other sites</string>
+    <!-- Description for accessing multiple domains permission. %1$s will be replaced by an integer indicating the number of additional hosts for which this webextension is requesting permission. -->
+    <string name="mozac_feature_addons_permissions_to_many_domains_description" tools:ignore="PluralsCandidate">Access your data on %1$d other domain</string>
+    <!-- Description for accessing multiple domains permission. %1$s will be replaced by an integer indicating the number of additional hosts for which this webextension is requesting permission. -->
+    <string name="mozac_feature_addons_permissions_to_many_domains_description_plural" tools:ignore="PluralsCandidate">Access your data on %1$d other domains</string>
```

Comment:
@Pike would you mind taking a look at these strings before landing? :)

Reply:
@Delphine might be able to help here instead.

Reply:
Thanks :)

Reply:
So I see a few things here:
* line 15-21: all comments are the same, so it's hard to understand how to localize these, and which string is supposed to be singular or plural
* Seems like they are assuming that all languages handle plurals like English?
* We unfortunately don't currently handle plural rules for Android
We should just do like we have usually done for these kinds of strings in the past, even though it's less than ideal (see the suggestions given here: https://github.com/mozilla-l10n/android-l10n/pull/11#discussion_r271636736)
thanks!

Reply:
> line 15-21: all comments are the same, so it's hard to understand how to localize these, and which string is supposed to be singular or plural

Sorry my bad, I assumed that add it `_plural` to the end of the entry will be sufficient, I will fix comments :)

> Seems like they are assuming that all languages handle plurals like English?

Yes, we have a entry for singular and another entry for plural. Should I be doing something extra?

> We unfortunately don't currently handle plural rules for Android
> We should just do like we have usually done for these kinds of strings in the past, even though it's less than ideal (see the suggestions given here: mozilla-l10n/android-l10n#11 (comment))
> thanks!

Thanks, the approach that we were using it is adding an entry for plural and singular, similar to what we are doing. I will update the commends to make it more noticeable.